### ISSUE TITLE
Release Twisted News 0.2.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#1746]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)| @radix reported|
|-|-|
|Trac ID|trac#1746|
|Type|task|
|Created|2006-05-25 01:10:37Z|

Or something

<details><summary>Searchable metadata</summary>

```
trac-id__1746 1746
type__task task
reporter__radix radix
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__news news
keywords__ 
time__1148519437000000 1148519437000000
changetime__1148520631000000 1148520631000000
version__ 
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="radix's avatar" src="https://avatars.githubusercontent.com/u/227068?s=50" width="50" height="50">](https://github.com/radix)|@radix set status to `closed`|
|-|-|

(In [16935]) Version and News updates from the release of Twisted News 0.2.0

Author: Release Automation, radix
Reviewer: exarkun, radix

Fixes [#1746](https://github.com/twisted/twisted/issues/1746)

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
